app/assets/javascripts/discourse/app/templates/wizard/step.gjs

```diff
@@ -32,6 +34,16 @@ export default RouteTemplate(
       return this.step.id === "ready";
     }
 
+    #goHomeOrQuickStart() {
+      if (this.siteSettings.bootstrap_mode_enabled) {
+        DiscourseURL.routeTo(
+          `/t/${this.siteSettings.admin_quick_start_topic_id}`
+        );
+      } else {
+        this.router.transitionTo("discovery.latest");
```

Comment:
Unrelated to your change, but this makes me think of something else: what if home isn't `discovery.latest`? Should we use `defaultHomepage` from `lib/utilities` instead?